### PR TITLE
Add etcd image

### PR DESCRIPTION
**Pull request by ereslibre**
This adds a very simple etcd image with the default package configuration.

### CODE REVIEW

File: derived_images/etcd/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM opensuse:leap
```
Comment (tboerger) [inline derived_images/etcd/Dockerfile:1]: No maintainer value?

Comment (ereslibre) [reply derived_images/etcd/Dockerfile:1]: Oops, thank you! Fixed.

File: derived_images/etcd/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+MAINTAINER SUSE Containers Team <containers@suse.com>
+
+RUN zypper ar -Gf http://download.opensuse.org/repositories/Virtualization:/containers/openSUSE_Leap_42.2/ containers
+
+RUN zypper ref && \
```
Comment (mssola) [inline derived_images/etcd/Dockerfile]: nit: both run statements can be merged

**Review by mssola — APPROVED** [review]
LGTM

File: derived_images/etcd/Dockerfile (hunk 2 above)
Comment (ereslibre) [reply derived_images/etcd/Dockerfile]: Fixed.

**Pull request merged**